extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -269,6 +269,18 @@ public String registerExtension(final String extensionName, final String path, f
         throws URISyntaxException, FalconException {
         Configuration conf = new Configuration();
         URI uri = new URI(path);
+        if (uri.getScheme() == null){
```

Comment:
I thought of doing  it..but later though it's better to explicitly tell the error with the path.Don't you think so as we are not saving anything .